src/_data/side-nav.yml

```diff
@@ -138,6 +119,8 @@
     - title: dart:html
       permalink: /libraries/dart-html
     - divider
+    - title: Iterable collections
+      permalink: /codelabs/iterables
     - title: Using streams
```

Comment:
**issue:** Move this whole block into **Development** and maybe make a heading for **Asynchronous Programming** and include these three plus the existing **Futures, async, await** page.

Reply:
The goal is to get rid of the **Development** section, just like we did on docs.flutter.dev, as it's too broad and there's no reason for a user to want to open it when looking for library related content like this.

While I don't think these articles follow our current standards, nor is the current set up of the **Core libraries** ideal yet, but I feel it's a more appropriate home while we continue to iterate.

Any concerns about keeping them here with that context?